### ISSUE TITLE
Doc Home / Title naming incorrect

### ISSUE THREAD

**Issue by michael-hawker** [MicrosoftDocs/CommunityToolkit#109]
![image](https://user-images.githubusercontent.com/24302614/170557403-dbb0ac35-d212-48f2-9591-fdc7723d1161.png)

These should say `Community Toolkits` in the sidebar and be `Community Toolkits Introduction` at the top, as shown when you visit one of the subpages like so:

![image](https://user-images.githubusercontent.com/24302614/170557992-07bb5bc9-2722-4a0f-9572-464229f9fff0.png)

Though the `.NET Community Toolkits` part here is extra and also incorrect?

The ".NET Community Toolkit" is a specific toolkit, so we should make sure to brand the collection of them as "Community Toolkits" to match our GitHub org or "CommunityToolkit".

I think there's 2 or 3 different things controlling all these different naming things, so I'm pretty confused on how we fix this...

It also probably doesn't help that we don't have all the content here to help solidify this, so maybe this has to wait to be completely fixed. Though it'd at least be nice to fix the ones showing on the landing page.

Related to #54 - I'll comment more in there about desired breadcrumb behavior.

**Comment by bijington** [comment]
@michael-hawker is this something I can assist with? Or at least learn about so I don't help to preserve the issue with future updates

**Comment by michael-hawker** [comment]
@bijington assistance would be great, I had tried to take a stab at this earlier but only got so far. I thought I had a link somewhere to more documentation from the docs side on how this all works, but I can't find it now.

@mattwojo any suggestions on what might be going on here or where we can learn more about breadcrumbs and the toc connections?

**Comment by bijington** [comment]
@michael-hawker I have no clue if this is in anyway correct or even helpful but here are some points I have noticed:

> These should say Community Toolkits in the sidebar and be Community Toolkits Introduction at the top, as shown when you visit one of the subpages like so:

I suspect that for some reason the title from the first child in the breadcrumbs file here: https://github.com/MicrosoftDocs/CommunityToolkit/blob/cbba7bbe83206dd4fd0c2a1d2093a3dc8419955d/docs/breadcrumb/toc.yml#L13 is being used in the breadcrumb for some reason. I actually think according to the breadcrumb file that we have, it should read `Docs/.NET/Community Toolkits` no Introduction.

Another interesting detail that I have noticed is that either the `index.md` filename appears to invoke some special behaviour or it is the first child of a section that doesn't appear to affect the breadcrumb display (see images).

## First child/index.md -> No Alerts in the breadcrumb
![first child](https://user-images.githubusercontent.com/17139988/172243096-d8c39da5-75cf-4f22-932f-da8068cdfded.png)

## Second child -> Alerts in the breadcrumb
![second child](https://user-images.githubusercontent.com/17139988/172243302-98cdefd3-ece0-40ba-9b12-af9b367becb3.png)

Finally the .NET MAUI Community Toolkit does appear to have a valid breadcrumb throughout apart from the behaviour I just mentioned.

**Comment by bijington** [comment]
Oh I have just noticed that your `introduction.md` file is in the `docs` folder whereas we have stuck ours a level down in the `maui` folder. Not sure this has any impact but I am just playing spot the difference 😄

**Comment by michael-hawker** [comment]
Yeah, the plan for the dotnet docs is to have the intro page at the root and subfolders for the various few packages there are (diagnostics, high performance, mvvm). As otherwise we'd have `docs.microsoft.com/dotnet/communitytoolkit/dotnet/mvvm` for instance which would be confusing instead of `docs.microsoft.com/dotnet/communitytoolkit/mvvm`.
